templates/tools/dockerfile/test/sanity/Dockerfile.template

```diff
@@ -48,9 +48,10 @@
   #======================================
   # More sanity test dependencies (bazel)
   RUN apt-get install -y openjdk-8-jdk
-  # TOOD(jtattermusch): pin the bazel version
+  # Revert Bazel to an old commit that supports running ./compile.sh 
+  # without a local protoc dependency
   RUN git clone https://github.com/bazelbuild/bazel.git /bazel
-  RUN cd /bazel && ./compile.sh
+  RUN cd /bazel && git reset --hard 19ece7c663cd99d338306e302e0f7ce023becfe3 && ./compile.sh
```

Comment:
Wouldn't `git checkout release-0.4.1` rely on Bazel not deleting that branch in their repo? They actually just deleted that branch today.